test/e2e/addonmanagement_test.go

```diff
@@ -44,9 +44,9 @@ var (
 	}
 )
 
-var _ = ginkgo.Describe("Enable addon management feature gate", ginkgo.Label("addon-manager"), func() {
+var _ = ginkgo.Describe("Enable addon management feature gate", ginkgo.Ordered, ginkgo.Label("addon-manager"), func() {
```

Comment:
why do we need the `ginkgo.Ordered` for this test?

Reply:
since we use the same addon name in each test case, so the two cannot run concurrently. It also help us to add a BeforeAll and AfterAll to enable/disable the feature gates, so we do not need to enable/disable for each case.